lua/r/utils.lua

```diff
@@ -163,6 +163,7 @@ end
 ---@param cmd string[] The command to execute.
 ---@param opts table Options.
 function M.system(cmd, opts)
+    opts = opts or {}
```

Comment:
Is adding this line sufficiently, or should we check `opts` every time an `opts` member is accessed?